### ISSUE TITLE
Less is more

### ISSUE THREAD

**Issue by Simsys** [larus-breeze/standards-larus-NMEA-protocol#3]
There are 18 repositories on larus breeze. That's not exactly clear. This repository contains a single file. Therefore I suggest to move this file to https://github.com/larus-breeze/documentation_and_utilities.

There you could also put the currently missing dok for the CAN protocol.

@MaxBaex What do you think about it?

**Comment by MaxBaex** [comment]
I agree, we should clean up the repository structure and make it more distinct!  

https://github.com/larus-breeze/standards-larus-NMEA-protocol shall be deleted and the contend shall be moved: https://github.com/larus-breeze/documentation_and_utilities    documentation_and_utilities is intended for overall project documentation and helper scripts. 

**Suggestion for repository naming rules. I think it would be great to have a very simple set of rules:**
- Start with a sw_, hw_ or doc_  prefix if it is software, hardware or documentation. 
- Follow the prefix with a component name: larus, sensor, frontend, [audio or sound?], emulator.   Use larus for overview repositories which are not only for a specific component.
- Add an optional postfix for additional information e.g. _rs for rust software,   _lib for own git submodules
What do you think?

**Purpose of repositories:**
https://github.com/larus-breeze/documentation_and_utilities <= give an overall project overview   (rename to doc_larus ?)
- https://github.com/larus-breeze/documentation_discussions    <= private internal documentation and discussions  (Rename to doc_larus_discussions?)
- https://github.com/larus-breeze/hw_sensor <= hardware design sensor
- https://github.com/larus-breeze/sw_sensor <= firmware to the sensor
- https://github.com/larus-breeze/sw_sensor_algorithms  <= vario algorithms   (Rename to  sw_algorithms_lib ?)
- https://github.com/larus-breeze/SIL_flight_sensor_emulator  <= Simulator  (Rename to  sw_emulator ?) 
- https://github.com/larus-breeze/sw_bluepill_sound_and_GPIO   <= audio software for hw_utility   (Rename to sw_audio?)
- https://github.com/larus-breeze/.github <= Profile page behind: https://github.com/larus-breeze    Needs a rework as documents are duplicated there.
- https://github.com/larus-breeze/hw_display <= own display hardware design  (rename to hw_frontend ?)
- https://github.com/larus-breeze/sw_frontend_rs <=  rename to (sw_display_rs?)  I would like to see the naming pattern [sw,hw]_[display,sensor,]_   so that it ist clear which software belongs to which hardware. 


**These repositories are stale:**  
- https://github.com/larus-breeze/sw_frontend  =>  Horst maintains his software via SVN. It is set private, so it can stay here. 
- https://github.com/larus-breeze/hw_sense  =>  Could be set private or moved to my private account ?
- https://github.com/larus-breeze/sys_angle_of_attack_sensor   => Andi W. Bachelor thesis.   Is in a research state and does not belong to the Larus core competencies. I think it should be moved to a different parent.  
- https://github.com/larus-breeze/sw_utility   =>  Just experimental software for an audio implementation. Shall we put it to a private repository? The current audio implementation is here: sw_bluepill_sound_and_GPIO     



My thoughts got more extensive than expected, but I think this is very relevant as an entrance point into our project.

**Comment by Simsys** [comment]
"My thoughts got more extensive than expected, but I think this is very relevant as an entrance point into our project." Yeah!

My view (+ means: good, I like it)
doc_larus? +
doc_larus_discussions? +
sw_algorithms_lib? +
sw_emulator? -> I think SIL should survive in name, proposal: sw_sil_emulator
sw_audio? -> Perhaps to general, proposal: sw_audo_box
hw_frontend? +
sw_display_rs? One line above, the proposal is frontend, display is perhaps to general

**Comment by MaxBaex** [comment]
This repository will be deleted soon. Discussion can continue here: https://github.com/larus-breeze/doc_larus_discussions/discussions/14

Moved the following Repositories from the larus-breeze organization as less is more:
https://github.com/larus-breeze/sys_angle_of_attack_sensor, https://github.com/larus-breeze/hw_sense, https://github.com/larus-breeze/sw_utility

Made the following repository private as it is not relevant for the public:
https://github.com/larus-breeze/hw_frontend_pcbadapter

Renamed these repositories: hw_display -> hw_frontend, documentation_and_utilities -> doc_larus, SIL_flight_sensor_emulator -> sw_emulator_sil, sw_sensor_algorithms -> sw_algorithms_lib, hw_utility -> hw_audio_box, sw_audio -> sw_audio_box, documentation_discussions -> doc_larus_discussions

Moved the NMEA Protocol Definition from https://github.com/larus-breeze/standards-larus-NMEA-protocol to https://github.com/larus-breeze/doc_larus/blob/master/documentation/Larus_NMEA_Protocol.md. I will delete the standards-larus-NMEA-protocol repository.

sw_sensor and sw_emulator_sil need a submodule update to reflect the naming change. Github redirects to new names so this is not immediately necessary.
